### PR TITLE
R doc 1835 ts document query

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown [hunk 1]
```diff
@@ -96,6 +150,24 @@ Learn more about `session.Query` [here](../../../../client-api/session/querying/
...
+### Document Query
+
+* A Document Query using only the `From()` method.  
+  {CODE TS_DocQuery_1@DocumentExtensions\TimeSeries\TimeSeriesTests.cs /}  
+
+* A Document Query using `Between()`.  
+  {CODE TS_DocQuery_2@DocumentExtensions\TimeSeries\TimeSeriesTests.cs /}  
+
+* Two Document Queries using `FromFirst()` and `FromLast()`. These return the first three 
+  days of the 'HeartRate' time series, and the last three days, respectively.
```
Comment (karmeli87) [inline Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown:162]: Heartrates and friends.

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+Where(Expression<Func<TimeSeriesEntry, TTag, bool>> predicate);
+{CODE-BLOCK/}
+
+| Parameter | Type | Description |
+| - | - | - |
+| **name** | `string` | The name of the time series (in one or more documents) to query |
+| **start** | `DateTime` | First parameter for `Between()`; the beginning of the time series range to filter. |
+| **end** | `DateTime` | Second parameter for `Between()`; the end of the time series range to filter. |
+| **timePeriod** | `Action<ITimePeriodBuilder>` | Expression returning a number of time units representing a time series range either at the beginning or end of the queried time series. |
+| `LoadByTag` type parameter | `TTag` | Time series entry tags can be just strings, but they can also document IDs, representing a reference to an entity. `LoadByTag` takes the type of the entity. |
```
Comment (aviv86) [inline Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown:85]:
```suggestion
| `LoadByTag` type parameter | `TTag` | Time series entry tags can be just strings, but they can also be document IDs, representing a reference to an entity. `LoadByTag` takes the type of the entity. |
```

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown]: fixed

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/document-extensions/timeseries/client-api/session/querying.markdown]: fixed

**Review by aviv86 — APPROVED** [review]

**Comment by ppekrol** [comment]
@morhilai can you rebase so I can merge?

**Review by karmeli87 — APPROVED** [review]

**Pull request merged**